.pre-commit-config.yaml

```diff
@@ -105,27 +132,28 @@ repos:
         entry: ./site.sh lint-css
         language: system
         files: \.scss$
+        exclude: _rst-content\.scss$
```

Comment:
I'm not sure if this line is still needed since we ignore this file in `.stylelintignore`